README.md

```diff
@@ -252,3 +252,65 @@ python3 -m twine upload dist/*
 
 ## Requirements
 Python 3.9 or newer
+
+# Optional Timeout Feature
+
+When deployed behind a server gateway, such as AWS API Gateway, the gateway may impose constraints
+on timeout. For example, AWS API Gateway has a timeout of 29 seconds.
+
+The ubkg-api (loaded as a library by the hs-ontology-api) can handle timeouts before they result in errors in a gateway. 
+The ubkg-api can return detailed explanations for timeout issues, instead of relying on the 
+sometimes ambiguous messages from the gateway (e.g., a HTTP 500).
+
+To enable custom management of timeout, specify values in the **app.cfg** file, as shown below.
+
+```commandline
+# Maximum duration for the execution of timeboxed queries to prevent service timeout, in seconds
+# The AWS API gateway timeout is 29 seconds.
+TIMEOUT=28
+```
+
+# Managing large payloads
+When deployed behind a server gateway, such as AWS API Gateway, the gateway may impose constraints
+on the size of response payload. For example, AWS API Gateway has a response payload limit of 10 MB.
+
+The ubkg-api (loaded as a library by the hs-ontology-api) can return a custom
+HTTP 413 error (Payload too Large) when the response exceeds a specified threshold.
```

Comment:
Since we've agreed to use 403 response, 413 should be updated here to avoid confusion.